code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -210,6 +213,11 @@ adjustFireLoss(-(maxHealth / 70 + 0.5 + (maxHealth / 70) * recovery_aura/2)*(m))
 								Z.recovery_new = leader_aura_strength
 
 /mob/living/carbon/Xenomorph/proc/handle_aura_receiver()
+	if(on_fire) //Burning xenos can't benefit from pheromones; 0 everything out, return.
```

Comment:
This would be more efficient to handle in `handle_aura_emiter()`.
If they are on fire they don't receive pheros, instead of receiving and losing them.

Reply:
Still need to make sure if they have them from a prior tick they get zeroed out, though yes, I'm adding preventative checks to emitter.

Reply:
Pheros get reset every tick. If you cut the source you cut the receiver as well.